openfermioncirq/gates/common_gates.py

```diff
@@ -110,6 +111,10 @@ class XXYYPowGate(cirq.EigenGate,
 
     `ofc.XXYY` is an instance of this gate at exponent=1.
     """
+    @deprecation.deprecated(deprecated_in='v0.3.0', removed_in='v0.4.0',
```

Comment:
Looks like 0.3.0 is already released and the current version is 0.4.0. Shouldn't that be deprecat_in=0.4.0, removed=0.5.0?

Reply:
Indeed. Thanks for catching! Fixed both decorators.